bin/boot

```diff
@@ -54,10 +84,11 @@ then
 else
     file=$ODOO_CONF
 fi
-if [ ! -f "$file" ]; then
-    cp /opt/sources/odoo.conf $file 
-    echo "Copy template as conf file $file"
+if [ -f "$file" ]; then
```

Comment:
Is it retro-compatible with our current infra (ADD odoo.conf in the Dockerfile)? Best is to give as much flexibility as possible. It should be possible to use a custom odoo.conf file (ideally that file could contain the `_ADMIN_PASSWORD_`, `_DB_USER_`, `_DB_PASSWORD_`, etc. variables)